nibabel/funcs.py

```diff
@@ -219,3 +219,69 @@ def _aff_is_diag(aff):
     ''' Utility function returning True if affine is nearly diagonal '''
     rzs_aff = aff[:3, :3]
     return np.allclose(rzs_aff, np.diag(np.diag(rzs_aff)))
+
+
+def crop_image(img, bounds, margin=0):
```

Comment:
It looks like the bounds only apply to the first three dimensions.   This are nearly always the spatial dimensions, but not so for Minc.

I'm not sure how margin interacts with bounds .   What do you get for bounds of [[2, -2], [2, -2], [-2, 2]] and margins of 2?

Do we need this function as well as the slicing functionality?

Reply:
I'm currently leaving this untouched as I work on the slicing. If it turns out this is no longer useful,I'll drop it.

But for your example, that would end up giving you the image back, untouched.

And yes, it doesn't work for Minc, at the moment.